carma_wm/test/CARMAWorldModelTest.cpp

```diff
@@ -992,14 +992,14 @@ TEST(CARMAWorldModelTest, getLaneletsFromPoint)
   // Create a complete map
   test::MapOptions mp(1,1);
   auto cmw_ptr = test::getGuidanceTestMap(mp);
-  auto underlyings = cmw_ptr->getLaneletsFromPoint({0.5,0.5});
+  auto underlyings = cmw.getLaneletsFromPoint({0.5,0.5});
   ASSERT_EQ(underlyings.size(), 1);
   ASSERT_EQ(underlyings.front().id(), 1200);
 
   auto ll_1500 = test::getLanelet(1500, {getPoint(0.0,0.1, 0),getPoint(0.0,1.1, 0)}, 
                          {getPoint(1.0,0.1, 0),getPoint(1.0,1.1, 0)}); // another lanelet the point is in
   cmw_ptr->getMutableMap()->add(ll_1500);
-  underlyings = cmw_ptr->getLaneletsFromPoint({0.5,0.5});
+  underlyings = cmw.getLaneletsFromPoint({0.5,0.5});
```

Comment:
Is this change to this test correct? Since the cmw variable is not provided to getGuidanceTestMap I don't see how these two lines could be equivalent.

Reply:
Good catch. I had to revert this back